tests/unit_tests/data_model/test_validate_data.py

```diff
@@ -695,3 +696,41 @@ def test_prepare_model_parameter():
     data_validator.data_dict["type"] = "int64"
     data_validator._prepare_model_parameter()
     assert isinstance(data_validator.data_dict["value"][0], int)
+
+
+def test_check_version_string(caplog):
+    data_validator = validate_data.DataValidator()
+
+    valid_versions = [
+        "1.0.0",
+        "0.1.0",
+        "2.3.4",
+        "1.0.0-alpha",
```

Comment:
Related to above question. So we really want to allow this kind of version structure?

Reply:
This is how semantic versioning is defined, see https://semver.org/

Reply:
Ok all good then.